.travis.yml

```diff
@@ -3,9 +3,4 @@ jdk:
   - openjdk8
 install: true
 script:
-# JS tests will fail because no headless chrome can be found, so we skip tests
-  - ./gradlew test -x jsBrowserTest
-  - ./gradlew publishToMavenLocal -x jsBrowserTest
-  - cd antlr-kotlin-examples-jvm && ../gradlew --info check && cd ..
-  - cd antlr-kotlin-examples-mpp && ../gradlew --info check && cd ..
-#  - cd antlr-kotlin-examples-js && ../gradlew --info check && cd ..
+  - ./ci.sh
```

Comment:
Good idea

Reply:
sometimes the travis build and sometimes the jitpack build was broken.. we can now just invoke ./ci.sh to check both. I already did this in PR #55 for PR #54.